openapi.yaml

```diff
@@ -928,15 +932,15 @@ components:
         referencePath:
           type: string
           description: Where to find the reference in the Bundle entry
-          example: 'provider.reference'
+          example: provider.reference
       description: A connection between entries in a Bundle
     ResourceRequest:
       properties:
-        resource:
-          type: object
+        resourceType:
+          type: string

```

Comment:
I don't think this is correct.
I think that `additionalProperties` must be specified to indicate that the response object has more keys besides `resourceType `

https://swagger.io/docs/specification/data-models/dictionaries/

Reply:
updated